### DISCUSSION TITLE
Does anyone know companies that can recycle/refurbish faulty memory modules?

### DISCUSSION THREAD

**Discussion by Nable80** [memtest86plus/memtest86plus#127] [General]
I'm sorry for this off-topic but I hope someone here can recommend something.

I have a lot of faulty DDR2/DDR3 ECC RAM modules (and few DDR4 ones). Most of them have just one faulty chip (I'm not even sure that ICs are bad, maybe it's just the usual problem of Pb-free soldering alloy) but I don't have proper diagnostic hardware and repairing them on my own would be too time consuming, I have a lot of other tasks as a sysadmin. At the same time I can't just throw them away: there are hundreds of large memory chips, 80-90% of which aren't bad. Can anyone recommend a company in China (I can send packages there) or Eastern Europe which can recycle/refurbish these modules, so they will be used somewhere instead of just decaying in trash and polluting the Earth?

I asked local repair companies in my city but they're not interested (i.e. it's cheaper to buy used working modules than to repair broken ones), then I asked my colleagues from China and they told me there were such companies in their province but many of them were closed or changed their business, so they couldn't recommend any.

**Comment by BenMyers43** [comment]
Try to find a company that is an e-cycler, a recycler of electronics.  An honest one will pay just short of $20/lb (454 grams).  Very little is done to repair RAM.  The memory sticks go into a smelter and the precious metals and rare earths are recycled.

I will guess from your writing style that you are in the United States, so finding an e-cycler near you should be easy.

**Reply by Nable80** [reply]
No, I'm not from US, I'm from Eastern Europe and e-cyclers here only pay for ancient electronic components which contain significant amount of precious metals. In order to recycle almost everything else, one have to pay _them_ for this work instead.
Thank you for reply anyway.

**Comment by Quarktwin** [comment]
as far as i know, there is a company call Quarktwin that  can recycle faulty memory modules,  but it is in china, i am not sure if the answer can help you. if you are interested in it. then just check out quarktwin.com.

**Reply by Nable80** [reply]
Thank you, I hope their services are available in Mainland China! I sometimes send parts to my colleagues there, so I'll ask them if they can contact your company and get any profit from this.
